src/pages/[language]/events/index.vue

```diff
@@ -55,6 +55,8 @@
 </template>
 
 <script setup>
+  definePageMeta({ layout: 'content-page' })
```

Comment:
If you don't mind too much could you find-and-replace all of these to add a semicolon? :sweat_smile: 
```suggestion
  definePageMeta({ layout: 'content-page' });
```